.github/workflows/linux_build.yml

```diff
@@ -1,8 +1,11 @@
 name: Linux builds
 
 on:
-  - push
```

Comment:
Looks like this is no longer running on pushes to `mas

Reply:
😭 sorry about that!